scripts/get_dbdeployer_mysql_json.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+# Bash script generating JSON entry
+# for dbdeployer's downloads list
+
+# v0.1
+
+# 2021-04-30 - lefred - v0.1
+
+
```

Comment:
We should check here:
* whether `html2text` exists
* whether the version is 2+ (version 1 has incomplete features)